### PR TITLE
AWSO docs update

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/observability/aws/integrations/amazon-elasticache.md [hunk 1]
```diff
@@ -56,19 +56,17 @@ The Amazon ElastiCache app uses the following logs and metrics:
...
 account=dev region=us-east-1 namespace=aws/elasticache metric=CacheHitRate statistic=Average CacheClusterId=* CacheNodeId=* | avg by account, region, namespace, CacheClusterId, CacheNodeId
-```
-
-### Sample Query (CloudTrail Log based)  
+``` 
 
 Top 10 ReplicationGroupId's:
 
-```sql
+```sql title="CloudTrail Log based"
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-elasticache.md:69]:
```suggestion
```sql title="CloudTrail log-based"
```

File: docs/observability/aws/integrations/amazon-sns.md [hunk 2]
```diff
@@ -70,18 +68,15 @@ account={{account}} region={{region}} namespace={{namespace}} "\"eventsource\":\
 | sort by _count, event_status asc
 ```
 
-
-### Sample Query (Metric based)
-
-```sql title="Messages Published"
+```sql title="Metric based - Messages Published"
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sns.md:71]:
```suggestion
```sql title="Metric-based - Messages Published"
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 3]
```diff
@@ -63,19 +63,13 @@ The App uses SQS logs and metrics for:
...
 ```
 
-
-
 ### Sample Query
 
-**Messages Received (Metric based)**:
-
-```
+```sql title="Messages Received (Metric based)"
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:68]:
```suggestion
```sql title="Messages Received (Metric-based)"
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 4]
```diff
@@ -63,19 +63,13 @@ The App uses SQS logs and metrics for:
...
 metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename
 ```
 
-**Top 10 users (CloudTrail Log based)**:
-
-```sql
+```sql title="Top 10 users (CloudTrail Log based)"
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:72]:
```suggestion
```sql title="Top 10 users (CloudTrail log-based)"
```

File: docs/observability/aws/integrations/aws-api-gateway.md [hunk 5]
```diff
@@ -63,18 +63,14 @@ The AWS API Gateway ULM app uses the following logs and metrics:
...
-```sql
+```sql title="Average Latency by API Name (Metric based)"
 _sourceCategory=Labs/AWS/APIGateway/Metric Namespace=aws/apigateway metric=Latency statistic=Average
 account=* region=* entity=* | avg by region, entity
 ```
 
-### Sample Query (CloudTrail Log based)
-
-```sql
+```sql title="CloudTrail Log based"
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-api-gateway.md:73]:
```suggestion
```sql title="CloudTrail log-based"
```

File: docs/observability/aws/integrations/aws-api-gateway.md [hunk 6]
```diff
@@ -88,47 +84,13 @@ _sourceCategory=Labs/AWS/CloudTrail/APIGateway "apigateway.amazonaws.com" Namesp
...
-   * **Access Key ID and Secret Access Key**. Enter your Amazon [Access Key ID and Secret Access Key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html).
-   * **Scan Interval**. Use the default of 5 minutes. Alternately, enter the frequency Sumo Logic will scan your S3 bucket for new data.
-   * **Enable Timestamp Parsing**. Select the check box.
-   * **Time Zone**. Select Ignore time zone from log file and instead use, and select UTC.
-   * **Timestamp Format.** Select Automatically detect the format.
-   * **Enable Multiline Processing**. Select the check box, and select Infer Boundaries.
-2. Click **Save**.
+## Viewing AWS API Gateway dashboards
 
+We highly recommend you view these dashboards in the [Explore View](../deploy-use-aws-observability/view-dashboards.md) of the AWS Observability solution.
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-api-gateway.md:89]:
```suggestion
We highly recommend you view these dashboards in the AWS Observability solution > [Explore View](../deploy-use-aws-observability/view-dashboards.md).
```

Comment (kimsauce) [inline docs/observability/aws/integrations/aws-api-gateway.md:89]:
```suggestion
We highly recommend you view these dashboards in the [Explore View](../deploy-use-aws-observability/view-dashboards.md) of our AWS Observability solution.
```

File: docs/observability/aws/integrations/aws-classic-load-balancer.md [hunk 7]
```diff
@@ -2,63 +2,42 @@
...
-types:
+The AWS Classic Load Balancer ULM app uses the following log and metric types:
 
 * Metrics included in the AWS/ELB namespace. For more details, see this [*AWS Services help*](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html) page. 
 
-The logs are stored in a .gzip format in the specified S3 bucket and
-contain these fields in this order: timestamp, elb client:port, backend:port, request_processing_time, backend_processing_time, response_processing_time, elb_status_code, backend_status_code, received_bytes, sent_bytes, request, user_agent, ssl_cipher, ssl_protocol
+* The logs are stored in a `.gzip` format in the specified S3 bucket and contain these fields in this order: timestamp, elb client:port, backend:port, request_processing_time, backend_processing_time, response_processing_time, elb_status_code, backend_status_code, received_bytes, sent_bytes, request, user_agent, ssl_cipher, ssl_protocol. For more details on the Classic Load Balancer Access log, see the  [Classic Load Balancer Access Logs](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/access-log-collection.html).
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-classic-load-balancer.md:18]:
```suggestion
* The logs are stored in a `.gzip` format in the specified S3 bucket and contain these fields in this order: timestamp, elb client:port, backend:port, request_processing_time, backend_processing_time, response_processing_time, elb_status_code, backend_status_code, received_bytes, sent_bytes, request, user_agent, ssl_cipher, ssl_protocol. For more details on the Classic Load Balancer Access log, see [Classic Load Balancer Access Logs](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/access-log-collection.html).
```

File: docs/observability/aws/integrations/aws-classic-load-balancer.md [hunk 8]
```diff
@@ -2,63 +2,42 @@
...
-
-This page provides examples and descriptions for each of the AWS
-Observability Classic Load Balancer pre-configured dashboards.
+## Viewing AWS Classic Load Balancer dashboards
 
-Elastic Load Balancing automatically distributes your incoming traffic
-across multiple targets, such as EC2 instances, containers, and IP
-addresses, in one or more Availability Zones. It monitors the health of
-its registered targets, and routes traffic only to the healthy targets.
+We highly recommend you view these dashboards in the [Explore View](../deploy-use-aws-observability/view-dashboards.md) of the AWS Observability solution.
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-classic-load-balancer.md:36]:
```suggestion
We highly recommend you view these dashboards in the [Explore View](../deploy-use-aws-observability/view-dashboards.md) of our AWS Observability solution.
```

File: docs/observability/aws/integrations/aws-dynamodb.md [hunk 9]
```diff
@@ -2,26 +2,21 @@
...
 
-The Sumo App for AWS Observability DynamoDB is a unified logs and
-metrics (ULM) App that provides operational insights into DynamoDB
-instances across your infrastructure. Preconfigured app dashboards allow
-you to monitor key metrics, view the throttle events, errors,
-latency and help you plan the capacity of DynamoDB instances in your
-environment.
+The Sumo Logic app for AWS Observability DynamoDB is a unified logs and metrics (ULM) app that provides operational insights into DynamoDB
+instances across your infrastructure. Preconfigured app dashboards allow you to monitor key metrics, view the throttle events, errors,
+latency and help you plan the capacity of DynamoDB instances in your environment.
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-dynamodb.md:12]:
```suggestion
The Sumo Logic app for AWS Observability DynamoDB is a unified logs and metrics (ULM) app that provides operational insights into DynamoDB instances across your infrastructure. Preconfigured app dashboards allow you to monitor key metrics, view the throttle events, errors, latency and help you plan the capacity of DynamoDB instances in your environment.
```

File: docs/observability/aws/integrations/aws-dynamodb.md [hunk 10]
```diff
@@ -78,28 +71,17 @@ _sourceCategory=Labs/AWS/DynamoDB account=* namespace=* "\"eventSource\":\"dynam
...
 ```
 
-## AWS DynamoDB Dashboards
-
-This page provides examples and descriptions for each of the AWS
-Observability DynamoDB pre-configured dashboards.
+## Viewing AWS DynamoDB dashboards
 
-[AWS DynamoDB](https://aws.amazon.com/dynamodb/) is a fast and flexible NoSQL database service that provides consistent, single-digit millisecond latency at any scale. 
+We highly recommend you view these dashboards in the [Explore View](../deploy-use-aws-observability/view-dashboards.md) of the AWS Observability solution.
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-dynamodb.md:76]:
```suggestion
We highly recommend you view these dashboards in the [Explore View](../deploy-use-aws-observability/view-dashboards.md) of our AWS Observability solution.
```

**Comment by jpipkin1** [comment]
FYI, I made an update to the app-install.md file in this PR with the "[Fix indenting of note in app-install.md](https://github.com/SumoLogic/sumologic-documentation/pull/3508/commits/e810f316acdc8bde6773a2b93505177977a83299)" commit.

File: docs/observability/aws/integrations/aws-network-load-balancer.md [hunk 11]
```diff
@@ -91,13 +73,7 @@ Use this dashboard to:
...
 ### 4. AWS Network Load Balancer - Reset (RST) Packets
 
-**The AWS Network Load Balancer - Reset (RST) Packets** dashboard
-provides detailed insights into the number reset (RST) packets received
-by the network load balancer. The dashboard shows the information for
-the total number of reset (RST) packets sent from a client to a target ,
-the total number of reset (RST) packets generated by the load balancer,
-and the total number of reset (RST) packets sent from a target to a
-client.
+The **AWS Network Load Balancer - Reset (RST) Packets** dashboard provides detailed insights into the number reset (RST) packets received by the network load balancer. The dashboard shows the information for the total number of reset (RST) packets sent from a client to a target , the total number of reset (RST) packets generated by the load balancer, and the total number of reset (RST) packets sent from a target to a client.
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-network-load-balancer.md:76]:
```suggestion
The **AWS Network Load Balancer - Reset (RST) Packets** dashboard provides detailed insights into the number reset (RST) packets received by the network load balancer. The dashboard shows the information for the total number of reset (RST) packets sent from a client to a target, the total number of reset (RST) packets generated by the load balancer, and the total number of reset (RST) packets sent from a target to a client.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting stuff

**Review by kimsauce — APPROVED** [review]

**Pull request merged**